app/models/partnering_organization.rb

```diff
@@ -0,0 +1,2 @@
+class PartneringOrganization < ApplicationRecord
+end
```

Comment:
pull from master and add the has_many relationship to resources

also, include a comment to detail the fields that this model has. look at this for an example: https://github.com/calblueprint/veterans360-web/blob/master/app/models/resource.rb